setup.py

```diff
@@ -11,6 +12,24 @@ def find_version():
                      re.MULTILINE).group(1)
 
 
+def dependencies():
+    deps = [
+        'bitstruct>=8.15.1',
+        'python-can>=3.3.4',
+        'textparser>=0.21.1',
+        'diskcache',
+        'argparse_addons',
+        'typing_extensions>=3.10.0.0',
+        'crccheck',
+    ]
+
+    if sys.platform == 'win32':
```

Comment:
There is syntax for making dependencies platform dependent. Python-can used `'windows-curses;platform_system=="Windows" and platform_python_implementation=="CPython"'`

However, i would avoid making the same mistake and instead add windows-curses as an optional dependency.
You would need to catch the ImportError on Windows and add a meaningful error message though.